use/adopter/setting-up-your-very-first-conversation.md

```diff
@@ -23,41 +23,47 @@
 *   Upload this XML from using this api.
 
     ```
-    curl --location --request POST 'http://localhost:9999/admin/v1/forms/upload' \
-    --header 'admin-token: EXnYOvDx4KFqcQkdXqI38MHgFvnJcxMS' \
-    --form 'form=@"{PATH_OF_ODK_FORM}"'
+    curl --location 'http://localhost:9999/admin/form/upload' \
+    --header 'admin-token: {ADMIN_TOKEN}' \
```

Comment:
Where is the admin token and other values going to come from?

Reply:
Hey the token is set while setting up UCI-APIs, in the env.